.eslintrc.js

```diff
@@ -0,0 +1,226 @@
+/* eslint-disable import/no-commonjs */
+/* eslint-disable no-undef */
+
+// `postcss-modules` lints css modules class names, but it currently crashes
+// eslint on vscode if you use webstorm or want to run the lint for the cli, you
+// can use this flag to enable it. This is set to true in CI
+const shouldLintCssModules =
+  process.env.LINT_CSS_MODULES === "true" || process.env.CI;
+
+module.exports = {
+  rules: {
+    strict: [2, "never"],
+    "no-undef": 2,
+    "no-var": 1,
+    "no-unused-vars": [
+      "error",
+      {
+        vars: "all",
+        args: "none",
+        varsIgnorePattern: "^_",
+        ignoreRestSiblings: true,
+      },
+    ],
+    "no-empty": [1, { allowEmptyCatch: true }],
+    "no-restricted-imports": [
+      "error",
+      {
+        paths: [
+          {
+            name: "moment",
+            message: "Moment is deprecated, please use dayjs",
+          },
+          {
+            name: "moment-timezone",
+            message: "Moment is deprecated, please use dayjs",
+          },
+        ],
+      },
+    ],
+    curly: [1, "all"],
+    eqeqeq: [1, "smart"],
+    "import/no-default-export": 2,
+    "import/no-named-as-default": 0,
+    "import/no-commonjs": 1,
+    "import/order": [
+      "error",
+      {
+        "newlines-between": "always",
+        alphabetize: {
+          order: "asc",
+          orderImportKind: "asc",
+          caseInsensitive: false,
+        },
+        groups: [
+          "builtin",
+          "external",
+          "internal",
+          "parent",
+          "sibling",
+          "index",
+        ],
+        warnOnUnassignedImports: false,
+      },
+    ],
+    "no-console": [2, { allow: ["warn", "error", "errorBuffer"] }],
+    "react/no-is-mounted": 2,
+    "react/prefer-es6-class": 2,
+    "react/display-name": 1,
+    "react/prop-types": 2,
+    "react/no-did-mount-set-state": 0,
+    "react/no-did-update-set-state": 0,
+    "react/no-find-dom-node": 0,
+    "react/no-children-prop": 2,
+    "react/no-string-refs": 2,
+    "react/no-unescaped-entities": 2,
+    "react/jsx-no-target-blank": 2,
+    "react/jsx-key": 2,
+    "react/forbid-component-props": [2, { forbid: ["sx"] }],
+    "react-hooks/exhaustive-deps": [
+      "warn",
+      { additionalHooks: "(useSyncedQueryString|useSafeAsyncFunction)" },
+    ],
+    "prefer-const": [1, { destructuring: "all" }],
+    "no-useless-escape": 0,
+    "no-only-tests/no-only-tests": [
+      "error",
+      {
+        block: [
+          "describe",
+          "it",
+          "context",
+          "test",
+          "tape",
+          "fixture",
+          "serial",
+          "Feature",
+          "Scenario",
+          "Given",
+          "And",
+          "When",
+          "Then",
+          "describeWithSnowplow",
+          "describeEE",
+        ],
+      },
+    ],
+    complexity: ["error", { max: 54 }],
+    ...(shouldLintCssModules
+      ? {
+          "postcss-modules/no-unused-class": "off",
```

Comment:
https://github.com/bmatcuk/eslint-plugin-postcss-modules/blob/master/lib/index.ts#L7-L9

so we basically add one rule

Reply:
Good point, done here: https://github.com/metabase/metabase/pull/41173/commits/e48a719e97b7821e516bfcb3e30f59714e222dfe

I had to manually add the plugin, but it looks like it doesn't need to be behind a flag. If the plugin is there but no rule is used eslint is smart enough it seems